### PR TITLE
Set up autonumeric docs

### PR DESCRIPTION
**Pull request by blackbaud-johnly**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-autonumeric?_sp.spa=pr-set-up-autonumeric-docs](https://developer.blackbaud.com/skyux-autonumeric?_sp.spa=pr-set-up-autonumeric-docs)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-autonumeric?_sp.spa=pr-set-up-autonumeric-docs](https://developer.blackbaud.com/skyux-autonumeric?_sp.spa=pr-set-up-autonumeric-docs)

File: src/app/app-extras.module.ts [hunk 1]
```diff
@@ -10,10 +10,25 @@ import {
...
-    SkyAutonumericModule
+    SkyAutonumericModule,
+    SkyDocsToolsModule
+  ],
+  providers: [
+    {
+      provide: SkyDocsToolsOptions,
+      useValue: {
+        gitRepoUrl: 'https://github.com/blackbaud/skyux-sample',
+        packageName: '@skyux/sample'
```
Comment (Blackbaud-SteveBrush) [inline src/app/app-extras.module.ts:29]: ☝️ Do these properties need to be adjusted to match Autonumeric?

File: src/app/public/modules/autonumeric/autonumeric-options-provider.ts [hunk 2]
```diff
@@ -7,12 +7,16 @@ import {
...
 /**
- * @description Provides options to the underlying autoNumeric library.
- * See: https://github.com/autoNumeric/autoNumeric
+ * Provides options to the underlying [autoNumeric library](https://github.com/autoNumeric/autoNumeric).
  */
 @Injectable()
 export class SkyAutonumericOptionsProvider {
 
+  /**
+   * Specifies a value that represents a settings object to pass to the autoNumeric jQuery plugin.
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/autonumeric/autonumeric-options-provider.ts:16]: The version of `autonumeric` that we use does not reference jQuery anymore.

File: src/app/public/modules/autonumeric/autonumeric.directive.ts [hunk 3]
```diff
@@ -61,6 +66,9 @@ const SKY_AUTONUMERIC_VALIDATOR = {
 })
 export class SkyAutonumericDirective implements OnInit, ControlValueAccessor, Validator {
 
+  /**
+   * Assigns the name of a property from `SkyAutonumericOptionsProvider`.
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/autonumeric/autonumeric.directive.ts:70]: I'm not sure this is correct for this version of autoNumeric. This input allows consumers to pass options to the underlying autoNumeric utility.

File: src/app/public/modules/autonumeric/autonumeric.directive.ts [hunk 4]
```diff
@@ -52,7 +52,12 @@ const SKY_AUTONUMERIC_VALIDATOR = {
 };
 // tslint:enable
 
-@Directive({
+  /**
+   * Wraps the autoNumeric jQuery plugin to format any type of number, including currency.
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/autonumeric/autonumeric.directive.ts:56]: jQuery shouldn't be mentioned. Also, it's not really a "plugin" anymore; more akin to a JavaScript "utility" or service.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
I created some rudimentary documentation a while ago for this, here: https://github.com/blackbaud/skyux-autonumeric#usage. Unsure if that's helpful.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-autonumeric?_sp.spa=pr-set-up-autonumeric-docs](https://developer.blackbaud.com/skyux-autonumeric?_sp.spa=pr-set-up-autonumeric-docs)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**